app/_locales/fr/messages.json

```diff
@@ -0,0 +1,609 @@
+  "lessThanMax": {
+    "message": "doit être inférieur ou égal à $1.",
+    "description": "helper pour la saisie hexadécimale en entrée décimale"
+  },
+  "limit": {
+    "message": "Limite"
+  },
+  "loading": {
+    "message": "Chargement..."
+  },
+  "loadingTokens": {
+    "message": "Chargement des jetons..."
+  },
+  "localhost": {
+    "message": "Localhost 8545"
+  },
+  "logout": {
+    "message": "Déconnexion"
+  },
+  "loose": {
+    "message": "Vacant"
+  },
+  "mainnet": {
+    "message": "Réseau principal Ethereum"
+  },
+  "message": {
+    "message": "Message"
+  },
+  "min": {
+    "message": "Minimum"
+  },
+  "myAccounts": {
+    "message": "Mes comptes"
+  },
+  "needEtherInWallet": {
+    "message": "Pour interagir avec des applications décentralisées à l'aide de MetaMask, vous aurez besoin d'Ether dans votre portefeuille."
+  },
+  "needImportFile": {
+    "message": "Vous devez sélectionner un fichier à importer.",
+    "description": "L'utilisateur doit ajouter un fichier pour continuer"
+  },
+  "needImportPassword": {
+    "message": "Vous devez entrer un mot de passe pour le fichier sélectionné.",
+    "description": "Mot de passe et fichier requis pour importer un compte"
+  },
+  "networks": {
+    "message": "Réseaux"
+  },
+  "newAccount": {
+    "message": "Nouveau compte"
+  },
+  "newAccountNumberName": {
+    "message": "Compte $1",
+    "description": "Nom par défaut du compte suivant à créer sur l'écran de création de compte"
+  },
+  "newContract": {
+    "message": "Nouveau contrat"
+  },
+  "newPassword": {
+    "message": "Nouveau mot de passe (min 8 caractères)"
+  },
+  "newRecipient": {
+    "message": "Nouveau destinataire"
+  },
+  "next": {
+    "message": "Suivant"
+  },
+  "noAddressForName": {
+    "message": "Aucune adresse n'a été définie pour ce nom."
+  },
+  "noDeposits": {
+    "message": "Aucun dépôt reçu"
+  },
+  "noTransactionHistory": {
+    "message": "Aucun historique de transaction."
+  },
+  "noTransactions": {
+    "message": "Aucune transaction"
+  },
+  "notStarted": {
+    "message": "Pas démarré"
+  },
+  "oldUI": {
+    "message": "Ancienne interface utilisateur"
+  },
+  "oldUIMessage": {
+    "message": "Vous êtes revenu à l'ancienne interface utilisateur.Vous pouvez revenir à la nouvelle interface via l'option dans le menu déroulant en haut à droite."
+  },
+  "or": {
+    "message": "ou",
+    "description": "choix entre la création ou l'importation d'un nouveau compte"
+  },
+  "passwordMismatch": {
+    "message": "les mots de passe ne correspondent pas",
+    "description": "dans le processus de création de mot de passe, les deux nouveaux champs de mot de passe ne correspondent pas"
+  },
+  "passwordShort": {
+    "message": "mot de passe pas assez long",
+    "description": "dans le processus de création de mot de passe, le mot de passe n'est pas assez long pour être sécurisé"
+  },
+  "pastePrivateKey": {
+    "message": "Collez votre seed phrase privée ici:",
```

Comment:
"Collez votre seed phrase privée ici:" -> "Collez votre clé privée ici:", there is an error here, it's not the seed but the private key.